src/Http/Http.Extensions/test/Microsoft.AspNetCore.Http.Extensions.Tests.csproj

```diff
@@ -4,17 +4,28 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <!-- This is needed to support codepaths that use NullabilityInfoContext. -->
     <Features>$(Features.Replace('nullablePublicOnly', '')</Features>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
Why do we need this? I thought this wasn't even used anymore.

Reply:
So we can test with reference assemblies, not runtime assemblies.